mc/Mongo-Core.package/MongoDatabase.class/instance/collectionAt..st

```diff
@@ -1,7 +1,5 @@
 accessing
 collectionAt: aString 
-	"Answers the collection in this database that has aString as name (or nil)"
+	"Answers a MongoCollection for this database that has aString as name (may not exist)."
 
```

Comment:
This was not necessary for this PR but it's part of a clean up on how to obtain collections from a db...